build/configs/rtl8730e/audio/defconfig

```diff
@@ -71,7 +72,7 @@ CONFIG_ARM_HAVE_NEON=y
 # CONFIG_ARM_FPU_ABI_SOFT is not set
 CONFIG_ARM_DPFPU32=y
 CONFIG_ARM_NEON=y
-# CONFIG_ARM_HAVE_MPCORE is not set
+CONFIG_ARM_HAVE_MPCORE=y
```

Comment:
oh please disable CONFIG_ARM_HAVE_MPCORE configuration,

please refer https://github.com/Samsung/TizenRT/commit/840fd41255b648b76412fb14871eace27205ede7

Reply:
I disabled this in all defconfigs